app/helpers/alchemy/admin/contents_helper.rb

```diff
@@ -13,14 +13,20 @@ def render_content_name(content)
         if content.blank?
           warning('Content is nil')
           return
-        else
-          content_name = content.name_for_label
         end
+
+        content_name = content.name_for_label
+
         if content.definition.blank?
           warning("Content #{content.name} is missing its definition")
-          title = Alchemy.t(:content_definition_missing)
-          content_name = %(<span class="warning icon" title="#{title}"></span>&nbsp;#{content_name}).html_safe
+
+          icon = content_tag :span, class: 'inline warning icon with-hint' do
+            content_tag :span, Alchemy.t(:content_definition_missing), class: 'hint-bubble'
+          end
```

Comment:
This is almost the same code as `PagesHelper#page_layout_missing_warning`, maybe make that a helper? Also, the added functionality has no spec... :(